packages/account-v2/src/containers/POICountrySelector/POICountrySelector.tsx

```diff
@@ -1,9 +1,10 @@
-import React, { useState } from 'react';
+import React, { Fragment, useState } from 'react';
 import { Button, Divider, InlineMessage, Text } from '@deriv-com/ui';
 import { CountrySelector } from '../../components/CountrySelector';
+import { API_ERROR_CODES, IDV_ERROR_CODES } from '../../constants';
 
 type TPOICountrySelectorProps = {
-    errorStatus?: string;
+    errorStatus: string | null;
```

Comment:
here then we could add typeof those config objects

Reply:
The errorStatus is returned after computation. So I need to change the default value in `checkIDVErrorStatus`. But I think being explicit as null or a string is better rather than string or undefined? What do u think?